### DISCUSSION TITLE
(How) does a YOURLS installation conflict with other installations?

### DISCUSSION THREAD

**Discussion by margomius** [YOURLS/YOURLS#2878] [Q&A] [answered]
Hi,
I registered a domain and want to host YOULRS on it.
Let's say it's called www.example.com
I'd intuitively install it in the path /yourls and access it via www.example.de/yourls/admin.
Can I then still host other html sites with index.html or subpages in /subpage/index.html?

Lets say I access this subpage with www.example.com/subpage
What happens when I create a Short-URL for /subpage which should be a conflict.

Also can I host a wordpress site next to Yourls?
My wordpress site would be the main folder and YOURLS would be in /yourls.

Any help is appreciated.

**Comment by dgw** [comment, marked as answer]
Start with the FAQ: https://yourls.org/#FAQ;htaccess

YOURLS needs its own `.htaccess` file, but if you're installing it in `/yourls/` that's not an issue. Note that you won't have short URLs like `somedoma.in/short`, but rather `somedoma.in/yourls/short`. YOURLS won't be called for anything outside `/yourls`, so you won't have a conflict. Even if it did, the sample `.htaccess` snippets you get with YOURLS will only check for a short URL if the path doesn't exist on the server.

**Reply by margomius** [reply]
Thank you for your fast reply and the link.

That's actually not as complicated as I anticipated.
So I won't install wordpress then because I want to have `somedoma.in/short`
But if i'm not mistaken I can create a simple html website (index.html) in the same directory as YOURLS?

Example:
Main Page (HTML/Bootstrap): `somedoma.in`
YOULRS Admin: `somedoma.in/admin`
Short-URL: `somedoma.in/short`

**Reply by dgw** [reply]
> But if i'm not mistaken I can create a simple html website (index.html) in the same directory as YOURLS?

Yes! Anything that actually exists on the filesystem will bypass YOURLS ☺

**Comment by minecraftchest1** [comment]
I have wordpress and yourls on the same machine (a raspberrypi4) under the same domain. I installed yourls under /short, and wordpress under /blog. I haven't had any issues.

**Reply by margomius** [reply]
That's what I am planning to do as well.

But just to be sure:

Your Short-URLs are 
`www.example.com/short/abc`
`www.example.com/short/xyz`

Rather than:
`www.example.com/abc`
`www.example.com/xyz`

Right?

**Reply by minecraftchest1** [reply]
Yes, that is correct.
